hedera-node/src/main/java/com/hedera/services/fees/calculation/schedule/txns/ScheduleCreateResourceUsage.java

```diff
@@ -0,0 +1,25 @@
+package com.hedera.services.fees.calculation.schedule.txns;
+
+import com.hedera.services.context.primitives.StateView;
+import com.hedera.services.fees.calculation.TxnResourceUsageEstimator;
+import com.hederahashgraph.api.proto.java.FeeData;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+import com.hederahashgraph.exception.InvalidTxBodyException;
+import com.hederahashgraph.fee.SigValueObj;
+
+public class ScheduleCreateResourceUsage implements TxnResourceUsageEstimator {
+    @Override
+    public boolean applicableTo(TransactionBody txn) {
+        return false;
+    }
+
+    @Override
+    public FeeData usageGiven(TransactionBody txn, SigValueObj sigUsage, StateView view) throws InvalidTxBodyException {
+        return null;
+    }
+
+    @Override
+    public long relativeLifetime(TransactionBody txn, long expiry) {
+        return 0;
+    }
```

Comment:
There is a default implementation in the `TxnResourceUsageEstimator`. No need to implement it. Same for the rest of the `ResourceUsage`-s